test/integration/testpaths/testpath_vmlc.py

```diff
@@ -176,6 +176,19 @@ def setUpClass(cls):
         cls.zone = get_zone(cls.apiclient)
         cls._cleanup = []
 
+        cls.skiptest = False
```

Comment:
This test does not require any special storage pool or with any tags. This check was put since it did not find any storage pool in UP state.i.e storage pool was present but was not in UP state.
So should I remove the check for skipTest?